Games/types/Mafia/roles/cards/WinWithMafia.js

```diff
@@ -20,7 +20,7 @@ module.exports = class WinWithMafia extends Card {
 
         // win by majority
         const hasMajority =
-          counts["Mafia"] + numBenandanteAlive >= aliveCount / 2 &&
+          counts["Mafia"] >= aliveCount / 2 &&
```

Comment:
@Golbolco  this is an unrelated bug in current master branch. numBenandanteAlive is not defined. modify at your will